pkg/runner/cluster_k8s.go

```diff
@@ -223,7 +227,16 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 			eg.Go(func() error {
 				defer func() { <-sem }()
 
-				return createPod(ctx, pool, podName, input, runenv, env, k8sConfig.Namespace, g, i)
+				err := createPod(ctx, pool, podName, input, runenv, env, k8sConfig.Namespace, g, i)
+
+				go func() {
```

Comment:
@raulk it might be a good idea to pick this information from Redis, rather than follow logs, as we discussed earlier. I have no idea if this will scale for 10k instances, but for now it is a solution - seems to work consistently with 1100 testplan instances.